common_utilities/Excel/excel_manage.py

```diff
@@ -0,0 +1,126 @@
+import openpyxl
+from openpyxl.reader.excel import load_workbook
+from openpyxl.workbook import Workbook
+from common_utilities.selenium.base_page import BasePage
+
+
+class ExcelManager(object):
+
+    def __init__(self, path):
+        self.path = path
+        print("Path", self.path)
+
+    def create_workbook(self):
+        wb = Workbook()
+        filepath = self.path
+        wb.save(filepath)
+
+    def create_sheet(self, sheet_name):
+        wb = load_workbook(self.path)
+        #sheet = wb.active
+        #sheet.title = sheet_name
+        #wb.create_sheet(index=1, title=sheet_name)
+        #wb.create_sheet(title=sheet_name)
+        wb.create_sheet(sheet_name)
```

Comment:
you can remove the commented codes if not needed for future.

Reply:
Changes made and pushed the new changes. Thank you !!

Reply:
I do not see the changes. I meant that you remove the commented code if they are not required. To clean the code.